packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -86,21 +94,40 @@ class ToOOXMLVisitor {
                     let textValueTag = TEXT_RULE(subNode.text);
 
                     let tag = TEXT_WRAPPER_RULE(propertyTag, textValueTag);
+
+                    if (isLinkPropertyPresent) {
+                        // some rels are already present
```

Comment:
You can just write 'relationships'. I see no need to abbreviate it.

```js
// Two elationships for numbering and style are already present
// and since we need to accommodate for link styles as well, we need a unique ID
// to represent them. Hence, 2 is added to offset the enumeration of `rId`.
```